src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -7,17 +8,24 @@ export default function FacilityAddress({
   name,
   facility,
   showDirectionsLink,
-  isHomepageRefresh,
-  clinicFriendlyName,
+  clinicName,
+  level = '4',
 }) {
   const address = facility?.address;
   const phone = facility?.telecom?.find(tele => tele.system === 'phone')?.value;
+  const extraInfoClasses = classNames({
+    'vads-u-margin-top--1p5': !!clinicName || !!phone,
+  });
+  const Heading = `h${level}`;
+  const HeadingSub = `h${parseInt(level, 10) + 1}`;
 
   return (
     <>
       {!!name && (
         <>
-          <strong>{name}</strong>
+          <Heading className="vads-u-font-family--sans vads-u-display--inline vads-u-font-size--base">
+            {name}
+          </Heading>
```

Comment:
Interesting trick with the heading. I didn't know that could be done.

Reply:
Yeah, it's neat. I believe someone picked it up from: https://medium.com/@Heydon/managing-heading-levels-in-design-systems-18be9a746fa3

It's used in the AlertBox component: https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/AlertBox/AlertBox.jsx#L74